### PR TITLE
Test for name of a multipart part set in Content-Disposition header

### PR DESCRIPTION
**Pull request by movepointsolutions**
I've coded down what I would like to see in my web attachments handling with mailio, regarding "name" attribute, currently mime::name_ is set to the value of "filename" Content-Disposition attr

### CODE REVIEW

**Comment by movepointsolutions** [comment]
#124

**Comment by movepointsolutions** [comment]
#123

**Comment by karastojko** [comment]
Let me check the RFC and implementation within *mailio*.

**Comment by movepointsolutions** [comment]
Look, I don't want to write unsupported changes, so please could you write a method to get "name" field of Content-Disposition header? I don't have any idea how it should be called, as 'name' is occupied by Content-Dispositon: filename="..."

**Comment by movepointsolutions** [comment]
for HTTP purposes!

**Comment by movepointsolutions** [comment]
Please take a look, I've noticed you've already had TODO regarding name() method

**Comment by movepointsolutions** [comment]
It seems I still can't use mailio for HTTP uploads, as line decoding policy breaks, even if I set VERYLARGE lines

**Comment by movepointsolutions** [comment]
image

File: test/test_message.cpp [hunk 1]
```diff
@@ -4148,6 +4148,7 @@ BOOST_AUTO_TEST_CASE(parse_multipart_content)
         "--my_bound\r\n"
         "Content-Type: text/html; charset=utf-8\r\n"
         "Content-Transfer-Encoding: Base64\r\n"
+        "Content-Disposition: form-data; name=\"name\"\r\n"
```
Comment (karastojko) [inline test/test_message.cpp:4151]: There is no `form-data` in the Content Disposition description in the RFC 2183. I would say it is related only to the HTTP specification. However, there is a MIME multipart subtype `form-data` which could be used for these purposes.

**Comment by karastojko** [comment]
> It seems I still can't use mailio for HTTP uploads, as line decoding policy breaks, even if I set VERYLARGE lines

The MIME format has the line length policy, but I think that HTTP does not. Thus, you cannot expect to mix these two approaches.

**Comment by karastojko** [comment]
No plans to mix HTTP with email protocols.

**Pull request closed without merging**